sycl/test-e2e/Graph/queue_state.cpp

```diff
@@ -0,0 +1,26 @@
+// REQUIRES: level_zero, gpu
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// Tests the return values from queue graph functions which change the
+// internal queue state.
+
+#include "graph_common.hpp"
+
+int main() {
+  queue Queue;
+
+  exp_ext::queue_state State = Queue.ext_oneapi_get_state();
```

Comment:
Rather a nit: this E2E test can probably be converted into a [unit test](https://github.com/intel/llvm/tree/sycl/sycl/unittests), because we shouldn't really need a device to query `queue` state, emulated `pi` layer should be enough.

Note: I'm not asking to rewrite this, or any other E2E tests to unit-tests, this is more of a FYI

Reply:
Good point! I've merged this test into the unit tests as it was fairly trivial.